model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -142,8 +143,7 @@ def __getitem__(self, key):
         elif key in self._input_to_verbose:
             return self._verbose[self._input_to_verbose[key]]
         else:
-            raise TritonModelAnalyzerException(
-                f"'{key}' Key not found in config")
+            raise KeyError
```

Comment:
Do we want to raise a key error if someone uses an improper argument to perf analyzer via `perf_analyzer_flags`?

Reply:
Good point. Will revert this change.